src/tests/ftest/util/dmg_utils.py

```diff
@@ -336,8 +337,24 @@ def storage_identify_vmd(self, uuid, verbose=False):
 
         """
         return self._get_json_result(
-            ("storage", "identify", "vmd"), uuid=uuid,
-            verbose=verbose)
+            ("storage", "led", "identify"), timeout=timeout,
+            reset=reset, ids=ids)
+
+    def storage_led_check(self, ids=None):
+        """Get the result of the 'dmg storage led check".
+
+        Args:
+            ids (list): Comma separated device id.
```

Comment:
list or str? Also, optional.
If str:
```suggestion
            ids (str, optional): Comma separated device id. Defaults to None.
```
Or, if list:
```suggestion
            ids (list, optional): List of device id. Defaults to None.
```

Reply:
Done